gpiozero/output_devices.py

```diff
@@ -503,6 +503,243 @@ def _blink_device(
+            'GS6': 1661.22,
+            'A6': 1760.0,
+            'AS6': 1864.66,
+            'B6': 1975.53,
+            'C7': 2093.0,
+            'CS7': 2217.46,
+            'D7': 2349.32,
+            'DS7': 2489.02,
+            'E7': 2637.02,
+            'F7': 2793.82,
+            'FS7': 2959.96,
+            'G7': 3135.96,
+            'GS7': 3322.44,
+            'A7': 3520.0,
+            'AS7': 3729.31,
+            'B7': 3951.07,
+            'C8': 4186.01,
+        }
+
+        self.midi_notes = {
+            """
+            Dictionary containing MIDI notes which can be played through the PWMBuzzer.
+            """
+            21: 27.50,
+            22: 29.14,
+            23: 30.87,
+            24: 32.70,
+            25: 34.65,
+            26: 36.71,
+            27: 38.89,
+            28: 41.20,
+            29: 43.65,
+            30: 46.25,
+            31: 49.0,
+            32: 51.91,
+            33: 55.00,
+            34: 58.27,
+            35: 61.74,
+            36: 65.41,
+            37: 69.30,
+            38: 73.42,
+            39: 77.78,
+            40: 82.41,
+            41: 87.31,
+            42: 92.50,
+            43: 98.0,
+            44: 103.83,
+            45: 110.0,
+            46: 116.54,
+            47: 123.47,
+            48: 130.81,
+            49: 138.59,
+            50: 146.83,
+            51: 155.56,
+            52: 164.81,
+            53: 174.61,
+            54: 185.0,
+            55: 196.0,
+            56: 207.65,
+            57: 220.0,
+            58: 233.08,
+            59: 246.94,
+            60: 261.63,
+            61: 277.18,
+            62: 293.66,
+            63: 311.13,
+            64: 329.63,
+            65: 349.23,
+            66: 369.99,
+            67: 392.0,
+            68: 415.30,
+            69: 440.0,
+            70: 466.16,
+            71: 493.88,
+            72: 523.25,
+            73: 554.37,
+            74: 587.33,
+            75: 622.25,
+            76: 659.26,
+            77: 698.46,
+            78: 739.99,
+            79: 783.99,
+            80: 830.61,
+            81: 880.0,
+            82: 932.33,
+            83: 987.77,
+            84: 1046.50,
+            85: 1108.73,
+            86: 1174.66,
+            87: 1244.51,
+            88: 1318.51,
+            89: 1396.91,
+            90: 1479.98,
+            91: 1567.98,
+            92: 1661.22,
+            93: 1760.0,
+            94: 1864.66,
+            95: 1975.53,
+            96: 2093.0,
+            97: 2217.46,
+            98: 2349.32,
+            99: 2489.02,
+            100: 2637.02,
+            101: 2793.82,
+            102: 2959.96,
+            103: 3135.96,
+            104: 3322.44,
+            105: 3520.0,
+            106: 3729.31,
+            107: 3951.07,
+            108: 4186.01,
+        }
+
+        def play(self, note):
+            """
+            PWMBuzzer function to play a given note from the two available dictionaries.
```

Comment:
It's a method not a function - but you can leave this part out. Start with "play a given note", specify the input options and ranges instead of "from the available dicts", and give two examples of usage. See other docstrings for format.

Reply:
Of course the `play` method disappears if we instead decide that setting a non-None value always turns the buzzer on.